include/boost/archive/codecvt_null.hpp

```diff
@@ -92,7 +92,7 @@ class BOOST_SYMBOL_VISIBLE codecvt_null<wchar_t> : public std::codecvt<wchar_t,
         return do_encoding();
     }
 public:
```

Comment:
Below: the body of the constructor is defined inline here and therefore does not need to be exported.